spec/models/active_shipping_calculator_spec.rb

```diff
@@ -15,15 +15,13 @@
 
   let(:carrier) { Spree::ActiveShipping::BogusCarrier.new }
```

Comment:
Since we've now fixed the `BogusCalculator` to properly create the `BogusCarrier`, can we remove this and do any stubbing directly on `calculator.carrier`? Then we can also remove all stubbing that returns this instance of the `carrier` on the `calculator`. This is just an idea, not sure if it will work.